kernel/isr-asm.s

```diff
@@ -168,4 +184,5 @@ pf_isr:
 enter_userspace:
 	mov %rdi,%rsp
 	pop_general_purpose_reg
+	add $8,%rsp  /* Skip over the status code. */
```

Comment:
can you not just ... pop?

Reply:
Pop requires a register to put the value in (and we're not MIPS, which has the special always-zero register)